src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -82,19 +92,25 @@ private Window AddTopLevelWindow()
             return win;
         }
 
-        private void AddMenu()
+        private void AddStatusBar()
         {
-            var menu = new MenuBar(new MenuBarItem []
-            {
-                new MenuBarItem("_Actions (F9)", 
-                    new MenuItem []
+            var statusBar = new StatusBar(
+                    _applicationData.OutputMode != OutputModeOption.None
+                    ? new StatusItem []
+                    {
+                        // Use Key.Unknown for SPACE with no delgate because ListView already
+                        // handles SPACE
+                        new StatusItem(Key.Unknown, "~SPACE~ Mark Item", null),
+                        new StatusItem(Key.Enter, "~ENTER~ Accept", () => Accept()),
```

Comment:
I just thought of another concern with `ENTER`... if the user types in the filter box, they might be inclined to hit `ENTER` to apply the filter. (especially if it doesn't filter-on-type which was a known issue in gui.cs)

I know I do this all the time in basically any search box.

I would hate for my ocgv to accidentally close from me wanting to apply the search.

Reply:
Alternative would be to have Ctrl+Enter to accept?

Reply:
I still prefer Ctrl+X for Exit (used by Nano) or Ctrl+Q for Quit (used by Micro), but ok waiting on user feedback.

Reply:
Again, this is how Windows (and Mac and I think Linux) GUI modal windows work. ENTER accepts and ESC cancels. 

Nobody is confused here:
![image](https://user-images.githubusercontent.com/585482/79698611-534c0000-8247-11ea-8561-f2dda2fe407c.png)

The filter is real-time, right? So there is no need for a user to hit ENTER to make it do it's thing, right?

One thing we could do, if this proves to be a problem (which I don't think it will be), is to make it so that if the filter `TextField` has focus, ignore ENTER (or move focus to the list). 

But I don't think this will be a problem because `Out-ConsoleGridView` is a modal.

Reply:
I suggest we ship it as-is and wait for user feedback. The model, as-is, is simple and matches an existing well established idiom.

Reply:
> especially if it doesn't filter-on-type which was a known issue in gui.cs

Tyler, do you know if there's an issue filed against `gui.cs` for this? I can't find one; happy to create and look into it regardless.

Reply:
Looks like it's fixed: https://github.com/migueldeicaza/gui.cs/issues/349 

If the filter Changed event starts working, then can you remove the apply button in this PR? I don't want any confusion with the `ENTER` key.

Reply:
Done. This is much better this way. 

Note there are bugs in the filter code though. I'll report them separately, but they have to do with filtering for single chars. Some seem to exist in `ogv` and some seem specific to the `ocgv` implementation...